packages/back-end/src/controllers/features.ts

```diff
@@ -143,6 +146,20 @@ async function getPayloadParamsFromApiKey(
 }
 
 export async function getFeaturesPublic(req: Request, res: Response) {
+  // Cloud only: Invoke GB for SSE rollout rules
+  let growthbook: GrowthBook | null = null;
+  if (IS_CLOUD) {
+    growthbook = new GrowthBook<AppFeatures>({
```

Comment:
Admittedly a bit inelegant. We could move this to app outside of the request/response lifecycle, but not entirely sure its worth it. Feedback welcome.